src/FixedDataTableColumn.js

```diff
@@ -164,18 +164,6 @@ class FixedDataTableColumn extends React.Component {
     */
    isReorderable: PropTypes.bool,
 
-   /**
-    * Whether cells in this column can be removed from document when outside
-    * of viewport as a result of horizontal scrolling.
-    * Setting this property to true allows the table to not render cells in
-    * particular column that are outside of viewport for visible rows. This
-    * allows to create table with many columns and not have vertical scrolling
-    * performance drop.
-    * Setting the property to false will keep previous behaviour and keep
-    * cell rendered if the row it belongs to is visible.
-    */
-   allowCellsRecycling: PropTypes.bool,
```

Comment:
Any idea if we could continue to support this?  We may want to try and find out how frequently used it is before removing.